.github/workflows/upload_all.yml

```diff
@@ -1,26 +1,132 @@
+        working-directory: platform/android
+      - name: Android - Set VERSION_NAME environment variable.
+        if: env.POLYPOD_UPLOAD_GOOGLE_PLAY == 'true'
+        run: echo "VERSION_NAME=$( ./getPackageInfo.sh /usr/local/lib/android/sdk/build-tools/30.0.0/aapt ./app/build/outputs/apk/release/app-release.apk versionName )" >> $GITHUB_ENV
+        working-directory: platform/android
+      - name: Android - Declare some variables to help label APK Upload
+        if: env.POLYPOD_UPLOAD_GOOGLE_PLAY == 'true'
+        id: vars
+        shell: bash
+        run: |
+          echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
+          echo "##[set-output name=sha_short;]$(echo ${GITHUB_SHA::7})"
+      - name: Android - Upload
+        # uses: r0adkll/upload-google-play@9745ef904e395471bca5696056a6ce8a60d18cf8
+        uses: polypoly-eu/upload-google-play@master
+        if: env.POLYPOD_UPLOAD_GOOGLE_PLAY == 'true'
+        with:
+          serviceAccountJsonPlainText: ${{ secrets.POLYPOD_ANDROID_STORE_SERVICE_ACCOUNT_JSON }}
+          packageName: coop.polypoly.polypod
+          releaseName: ${{ env.VERSION_CODE }} (${{ env.VERSION_NAME }}) - ${{ steps.vars.outputs.branch }}@${{ steps.vars.outputs.sha_short }}
+          releaseFiles: ./platform/android/app/build/outputs/apk/release/app-release.apk
+          track: internal
+          # whatsNewDirectory: distribution/whatsnew
+          # mappingFile:
 
-  call_upload_app_store:
-    name: Upload to the App Store
-    uses: ./.github/workflows/upload_app_store.yml
-    secrets:
-      POLYPOD_POLYPEDIA_REPORT_URL: ${{ secrets.POLYPOD_POLYPEDIA_REPORT_URL }}
-      POLYPOD_POLYPEDIA_REPORT_AUTHORIZATION: ${{ secrets.POLYPOD_POLYPEDIA_REPORT_AUTHORIZATION }}
-      POLYPOD_APPLE_IOS_APPSTORE_P12_FILE_BASE64: ${{ secrets.POLYPOD_APPLE_IOS_APPSTORE_P12_FILE_BASE64 }}
-      POLYPOD_APPLE_IOS_APPSTORE_P12_PASS: ${{ secrets.POLYPOD_APPLE_IOS_APPSTORE_P12_PASS }}
-      POLYPOD_APPLE_IOS_APPSTORE_PROVISION_PROFILE_BASE64: ${{ secrets.POLYPOD_APPLE_IOS_APPSTORE_PROVISION_PROFILE_BASE64 }}
-      POLYPOD_APPLE_IOS_APPSTORE_API_PRIVATE_KEY_BASE64: ${{ secrets.POLYPOD_APPLE_IOS_APPSTORE_API_PRIVATE_KEY_BASE64 }}
-      POLYPOD_APPLE_IOS_APPSTORE_API_KEY_ID: ${{ secrets.POLYPOD_APPLE_IOS_APPSTORE_API_KEY_ID }}
-      POLYPOD_APPLE_IOS_APPSTORE_API_ISSUER_ID: ${{ secrets.POLYPOD_APPLE_IOS_APPSTORE_API_ISSUER_ID }}
+      # iOS
+      - name: iOS - Lock Xcode version
+        if: env.POLYPOD_UPLOAD_APP_STORE == 'true'
+        working-directory: platform/ios
+        run: make lockxcodeversion
+      - name: iOS - Tests
+        if: env.POLYPOD_UPLOAD_APP_STORE == 'true'
+        working-directory: platform/ios
+        run: make test
+      - name: iOS - Install distribution prerequisites
+        if: env.POLYPOD_UPLOAD_APP_STORE == 'true'
+        working-directory: platform/ios
+        run: make install_distribution_prerequisites
```

Comment:
`run` is usually the last key, but it does not really matter.

Reply:
Moved it down everywhere! I <3 consistency.